lib/engine/game/g_1848/game.rb

```diff
@@ -701,36 +713,59 @@ def market_share_limit(corporation = nil)
         def can_take_loan?(entity)
           entity.corporation? &&
             entity.loans.size < maximum_loans(entity) &&
-            @loans.any?
+            @loans.any? && @take_out_loan_triggered
```

Comment:
Best practice is to follow the style used in the rest of the logic block...each check on its own line in this case